bitnami/opensearch/templates/_helpers.tpl

```diff
@@ -422,7 +422,7 @@ Return true if an authentication credentials secret object should be created
 Return the Opensearch authentication credentials secret name
 */}}
 {{- define "opensearch.secretName" -}}
-{{- default (include "common.names.fullname" .) .Values.security.existingSecret  -}}
+{{- include "common.tplvalues.render" (dict "value" .Values.security.existingSecret "context" $) | default (include "common.names.fullname" .) -}}
```

Comment:
Could you please use this approach instead? I would prefer keeping the `default` as it is or replace it with `if/else` as in other charts if its content gets too long:
```suggestion
{{- default (include "common.names.fullname" .) (tpl .Values.security.existingSecret $) -}}
```

In this case, because only 'string' values are expected, we can use `tpl` directly, similar to other charts:
https://github.com/bitnami/charts/blob/1c7dfc16b41dd75f08aa261efb74fa302ec9f2f4/bitnami/wordpress/templates/_helpers.tpl#L70-L79

Reply:
Updated.